content/sensu-core/1.8/migration.md

```diff
@@ -0,0 +1,380 @@
+| ------------- | ------------- |
+`filters: occurrences` | The built-in occurrences filter in Sensu Core 1.x is not provided in Sensu Go, but you can replicate its functionality using [this filter definition][10]. <!-- TODO: replace with link to filter asset guide -->
+`type: transport` | Transport handlers are no longer supported by Sensu Go, but you can create similar functionality using a pipe handler that connects to a message bus and injects event data into a queue.
+`filters: check_dependencies` | Sensu Go does not include a built-in check dependencies filter.
+`severities` | Severities are not available in Sensu Go.
+`handle_silenced` | Silencing is now disabled by default in Sensu Go and must be enabled explicitly using the built-in [`not_silenced` filter][15].
+`handle_flapping` | All check results are now considered events and are processed by event handlers.
+
+### 5. Upload your config to your Sensu Go instance
+
+Once you've reviewed your translated configuration, made any necessary updates, and added resource definitions for any filters and entities you want to migration, upload your Sensu Go config using sensuctl.
+
+{{< highlight shell >}}
+sensuctl create --file /path/to/config.json
+{{< /highlight >}}
+
+_PRO TIP: `sensuctl create` (and `sensuctl delete`) are powerful tools to help you manage your Sensu configs across namespaces. See the [sensuctl reference][5] for more information._
+
+You can now access your Sensu Go config using the [Sensu API][61].
+
+{{< highlight shell >}}
+# Set up a local API testing environment by saving your Sensu credentials
+# and token as environment variables. Requires curl and jq.
+export SENSU_USER=admin && SENSU_PASS=P@ssw0rd!
+
+export SENSU_TOKEN=`curl -XGET -u "$SENSU_USER:$SENSU_PASS" -s http://localhost:8080/auth | jq -r ".access_token"`
+
+# Return list of all configured checks
+curl -H "Authorization: Bearer $SENSU_TOKEN" http://127.0.0.1:8080/api/core/v2/namespaces/default/checks
+
+# Return list of all configured handlers
+curl -H "Authorization: Bearer $SENSU_TOKEN" http://127.0.0.1:8080/api/core/v2/namespaces/default/handlers
+{{< /highlight >}}
+
+You can also access your Sensu Go configuration in JSON or YAML formats using sensuctl; for example: `sensuctl check list --format json` Run `sensuctl help` to see available commands. For more information about sensuctl's output formats (`json`, `wrapped-json`, and `yaml`), see the [sensuctl reference][60].
+
+## Activate assets
+
+[Assets][12] are shareable, reusable packages that make it easy to deploy Sensu plugins.
+Although not required to run Sensu Go, we recommend [using assets to install plugins][45] where possible.
+For plugins that aren't available as assets, install outside of opt/sensu.
+
+[Sensu Plugins][21] in Ruby can still be installed via sensu-install by installing [sensu-plugins-ruby][20]; see the [installing plugins guide][22] and the [Sensu Plugins README][46] for more information.

```

Comment:
Okay we definitely need to call out the conflict between sensu core packages and sensu-plugins-ruby here.
They both provide sensu-install.  So if you are doing side by side, and you want to decommision your sensu core packages and switch to sensu-plugins-ruby.. you'll have to remove sensu package first and then install sensu-plugins-ruby because of the conflict.    You need to then re-install the plugins into the new environment.

Reply:
Added comment about re-installing sensu-install